docs/csharp/tuples.md

```diff
@@ -153,7 +153,8 @@ Beginning with C# 7.3, tuple types support the `==` and `!=` operators. These op
 
 [!code-csharp[TupleEquality](../../samples/snippets/csharp/tuples/tuples/program.cs#Equality "Testing tuples for equality")]
 
-There are several rules that make tuple equality tests more convenient. Tuple equality performs [lifted conversions](/dotnet/csharp/language-reference/language-specification/conversions.md#lifted-conversion-operators) if one of the tuples is a nullable tuple, as shown in the following code:
+There are several rules that make tuple equality tests more convenient. Tuple equality performs [lifted conversions](/dotnet/csharp/language-reference/language-specification/conversions#lifted-conversion-operators) if one of the tuples is a nullable tuple, as shown in the following code:
```

Comment:
That does fix the broken link but would you be so kind to change this to a relative url so the link gets validated during our build process?

/dotnet/csharp/language-reference/language-specification/conversions#lifted-conversion-operators -> ./language-reference/language-specification/conversions.md#lifted-conversion-operators